SYCL/ESIMD/api/unary_ops_heavy.cpp

```diff
@@ -172,14 +174,20 @@ int main(void) {
   passed &= test<float, 32>(mod_ops, q);
   passed &= test<double, 7>(mod_ops, q);
 
-  auto singed_ops = esimd_test::OpSeq<UnOp, UnOp::minus, UnOp::plus>{};
-  passed &= test<char, 7>(singed_ops, q);
-  passed &= test<short, 7>(singed_ops, q);
-  passed &= test<int, 16>(singed_ops, q);
-  passed &= test<int64_t, 16>(singed_ops, q);
-  passed &= test<half, 16>(singed_ops, q);
-  passed &= test<float, 16>(singed_ops, q);
-  passed &= test<double, 16>(singed_ops, q);
+  auto signed_ops = esimd_test::OpSeq<UnOp, UnOp::minus, UnOp::plus>{};
+  passed &= test<char, 7>(signed_ops, q);
+  passed &= test<short, 7>(signed_ops, q);
+  passed &= test<int, 16>(signed_ops, q);
+  passed &= test<int64_t, 16>(signed_ops, q);
+  passed &= test<half, 16>(signed_ops, q);
+  passed &= test<float, 16>(signed_ops, q);
+  passed &= test<double, 16>(signed_ops, q);
+
+#ifdef USE_BF16
```

Comment:
The obvious downside of this approach is the duplication of checks/tests.
I.e. on PVC the lines 178-184 started twice: 1st time as part of unary_ops_heavy.cpp test, and then (the same code + line 189) as part of unary_ops_heavy_pvc.cpp

Reply:
Right, good catch! I will add proper test selection.